### DISCUSSION TITLE
Is it me or this version is worse than the last one

### DISCUSSION THREAD

**Discussion by shadyx49** [namidaco/namida#328] [General]
I noticed the new update which was all about implementing youtube with the sign in feature so I deleted the old version and installed the new one only to notice that youtube is limited by a patreon paywall which makes me wonder if this version is better or worse than the last one which had yt working regularly without the sign in feature.

**Comment by MSOB7YY** [comment]
only you. maybe u missed reading the "**warning read  before installing**" section twice? + missed the 2 earlier messages on tg talking about this?

everything that was working before are still working. nothing was restricted. the patreon is for **new** actions only which require account.

things like search, viewing playlist, channels, etc are still the same.
even not all account actions require membership, i let "listing user playlists", "get playlists for a video", "mark video watched" and "personalized search".. so yeah

if you perhaps mean the home page, then it wasn't a home page before but "trending" page.

thats for the youtube side. many other things were added/improved u can check the changelog for that release

again, anything that was accessible before is still accessible. the patreon is for some new account actions. hope u get it
